src/plugins/home/server/services/tutorials/lib/tutorials_registry_types.ts

```diff
@@ -25,6 +25,7 @@ export type Platform = 'WINDOWS' | 'OSX' | 'DEB' | 'RPM';
 
 export interface TutorialContext {
   kibanaBranch: string;
+  staticAssets: IStaticAssets;
   [key: string]: unknown;
```

Comment:
Same, adding `staticAssets` to the tutorial context so that providers can use it.